packages/integration-test-app/windows/integrationtest/TestCommandResponse.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "TestCommandResponse.h"
+
```

Comment:
```suggestion
#include "TestCommandResponse.h"
#include <winrt/Windows.Data.Json.h>
```

Reply:
Json is already included in the header, since Json types are exposed in header.